### ISSUE TITLE
Help regarding training for new subject and meanings of constants in training code.

### ISSUE THREAD

**Issue by alakhsharmaa** [nihaomiao/WACV23_TSNet#16]
Hi, I have a few questions regarding the training code it would be great if you could help me.

1. **[Full Training]** The paper's and code's parameters do not match. which parameters we should use to train fully? we want to replicate your face.pth model which you share on Google Drive. can you please give me those parameters?

2. **[Fine Tuning]** Can you please also tell me the optimal hyper-parameters for the new subject? I'm training on the parameters mentioned in the code and the output is not that good on the new subject.

Thanks

**Comment by alakhsharmaa** [comment]
@nihaomiao it would be great if you could help me with these issues. thanks.

**Comment by nihaomiao** [comment]
Hi, @alakhsharmaa, thanks a lot for your interest in our work! The inconsistency of the parameter settings between paper and code is because I used different machines. In our paper, we used a machine that supports a larger batch size. I think you can just follow the parameter setting in the code. For fine-tuning, I have actually not tried on the new person before. But I think a small learning rate and the frames of the new person with different poses should be able to work.

**Comment by alakhsharmaa** [comment]
Thanks, @nihaomiao for the response. In the paper, you mentioned 30 subsequent frames or 150 videos for training. 
However, I observed all videos have straight faces in the first few how did the model learn different poses? are you selecting 30 frames from the random locations(I mean randomly selecting the first frame and then selecting 30 continuous frames) in the video?

Additionally, could you provide information about the specific parameters utilized during the training of the model you've shared via this [Google Drive link](https://drive.google.com/file/d/1tKZ0i1z-iEEQjRzXKOZNk5xVYvtxnfTc/view?usp=share_link)? Specifically, I'm interested in understanding the configuration and settings used for training on face forensic data.

**Comment by nihaomiao** [comment]
Hi, @alakhsharmaa, for video frame selection, I selected 30 continuous frames from the random locations so the training video may not have large pose changes. But I still think the video containing various poses will better help the model learn the subject appearance. For the released model, I don't remember the details very clearly, but the parameters given in our released code should produce a model with similar performance.

**Comment by alakhsharmaa** [comment]
Thanks @nihaomiao
